app.js

```diff
@@ -413,3 +413,63 @@ select_to.addEventListener('change', load_diff);
 
 fill_select_boxes(get_url_query_param('from'), get_url_query_param('to'));
 load_diff();
+
+
+function search_first_type(ppok, srqyr) {
```

Comment:
What do these names and `avr` mean?

Reply:
i am not good at naming variables, just had an idea 😅

Reply:
At the very least, we should use the name of the thing they are or hold (for example `diff` holds a difference). It is very confusing otherwise.